Card.java

```diff
@@ -49,39 +50,60 @@ public boolean getErrorFlag()
          return errorFlag;
      }
 
-      //Mutators 
-      public boolean set(char value, Suit suit)
-      {
-          if(isValid(value, suit))
-          {
-              this.value = value;
-              this.suit = suit;
-              errorFlag = false;
-              return true;
-          }
-          erroFlag = true;
-          return false;
-      }
+    //Mutators 
+    public boolean set(char value, Suit suit)
+    {
+        if(isValid(value, suit))
+        {
+            this.value = value;
+            this.suit = suit;
+            errorFlag = false;
+            return true;
+        }
+        erroFlag = true;
+        return false;
+    }
 
-      //Valid card data returned 
-      public String toString()
-      {
-          //if statement with errorFlag as parameter
-          //else return value + 'of' + suit string 
-      }
+    //Valid card data returned 
+    public String toString()
+    {
+        if(errorFlag)
+            return "Invalid Card";
+        return value + "of" + suit;
```

Comment:
Did this print out correctly? I think we may just need to add spaces before and after "of"

Reply:
Yes. It has been fixed and pushed back into my branch